newdle/client/src/components/summary/SummaryPage.js

```diff
@@ -1,4 +1,5 @@
 import React, {useState, useCallback} from 'react';
+import Switch from '@material-ui/core/Switch';
```

Comment:
I'm very surprised this works since we don't use `material-ui` in newdle. Maybe something pulling it in a a dependency (WTF).

Anyway, please use components from `semantic-ui-react` since that's our UI framework in newdle